src/wallet/wallet.cpp

```diff
@@ -2737,11 +2738,15 @@ void CWallet::DecrementNoteWitnesses(const Consensus::Params& consensus, const C
     if (consensus.NetworkUpgradeActive(pindex->nHeight, Consensus::UPGRADE_NU5)) {
         // pindex->nHeight is the height of the block being removed, so we rewind
         // to the previous block height
-        uint32_t blocksRewound{0};
+        uint32_t uResultHeight{0};
         assert(pindex->nHeight >= 1);
-        assert(orchardWallet.Rewind(pindex->nHeight - 1, blocksRewound));
-        assert(blocksRewound == 1);
```

Comment:
Is there any use for asserting that `blocksRewound` is 0 or 1?

Reply:
`blocksRewound` doesn't exist anymore; we always check against the explicit result height on line 2744.

Reply:
Oh yeah duh, not sure what I was thinking.